ios/ReactNativePageView.h

```diff
@@ -7,8 +7,11 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface ReactNativePageView: UIView <UIPageViewControllerDataSource, UIPageViewControllerDelegate,UIScrollViewDelegate>
 
+- (instancetype)initWithEventDispatcher:(RCTEventDispatcher *)eventDispatcher;
+
 @property(strong, nonatomic, readonly) UIPageViewController *reactPageViewController;
 @property(strong, nonatomic, readonly) UIPageControl *reactPageIndicatorView;
+@property(strong, nonatomic, readonly) RCTEventDispatcher *eventDispatcher;
```

Comment:
WDYT about make this property weak ? 

https://github.com/facebook/react-native/blob/84f5ebe4f941e752ba96b4d4a75dc6c6f5faa665/React/Base/RCTTouchHandler.m#L29

https://github.com/facebook/react-native/blob/8131b7bb7b4794e0e7003a6e3d34e1ebe4b8b9bc/Libraries/Text/TextInput/RCTBaseTextInputView.m#L25

Reply:
Ohh weakSelf is used https://github.com/react-native-community/react-native-viewpager/pull/46/files#diff-42352f0da98b553d1ace487a18f6d823R363. I close this comment.

Reply:
yes, sorry!